app/templates/events/view.hbs

```diff
@@ -1,6 +1,38 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
-  {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="ui grid">
+    <div class="row">
+      <div class="eleven wide column">
+        <h2 class="ui header">{{t 'Fossasia Test'}}</h2>
+      </div>
+      <div class="five wide column">
+        <button class="ui button small">{{t 'Preview'}}</button>
```

Comment:
<img width="372" alt="screen shot 2017-05-26 at 11 21 55 am" src="https://cloud.githubusercontent.com/assets/2404372/26482589/d7f478bc-4205-11e7-9d9f-b653075fa2c2.png">

The button sizes are non-uniform. Please pay attention to detail.

--- 

<img width="1680" alt="screen shot 2017-05-26 at 11 23 18 am" src="https://cloud.githubusercontent.com/assets/2404372/26482606/f0e21b2c-4205-11e7-879f-295fc976b4bc.png">

Why the extra space to the right ?